volatility3/framework/plugins/windows/modules.py

```diff
@@ -42,6 +42,12 @@ def get_requirements(cls) -> List[interfaces.configuration.RequirementInterface]
                 default=False,
                 optional=True,
             ),
+            requirements.StringRequirement(
+                name="name",
+                description="module name/sub string",
+                optional=True,
+                default="",
```

Comment:
Could you please use `None` rather than `""`?  They should both fail the check, but it's then easier to differentiate if the user provided a value or not

Reply:
Fixed :)